readme.txt

```diff
@@ -89,6 +89,7 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 - Dev: Add support for running php unit tests in PHP 8. #6678
 - Fix: Remove no-reply from inbox notification emails #6644
 - Performance: Avoid updating customer info synchronously from the front end. #6765
+- Feature: Add recommended payment methods in payment stetings. #6760
```

Comment:
Hi @louwie17,
Just chiming because I think that there is a typo here: `stetings`.